networkx/algorithms/bipartite/tests/test_cluster.py

```diff
@@ -61,6 +61,14 @@ def test_average_path_graph():
     assert bipartite.average_clustering(G, mode="min") == 1
 
 
+def test_select_nodes():
+    G = nx.bipartite.random_graph(5, 5, 0.5, seed=1)
+    total = [bipartite.average_clustering(G, nodes=[i], mode="min") for i in range(5)]
```

Comment:
I'm a bit confused here, in the docs it states that `average_clustering` for bipartite graphs should only be calculated with either the whole graph or bipartite sets. But here we can just use any iterable of nodes to get an clustering coefficient.

`average_clustering` uses `latapy_clustering` and in the docs of `latapy_clustering` there is no such limitation mentioned for the node iterable. Should we update our docs?